### PR TITLE
Pass over all Rails 5 warnings

### PR DESCRIPTION
**Pull request by vipulnsward**
Pass over all Rails 5 warnings, to make sure:
- we are ending sentences properly
- fixing of space issues
- fixed continuity issues in some sentences.

This is continuation of https://github.com/rails/rails/pull/24485  and takes care of warnings at all places.

### CODE REVIEW

**Comment by vipulnsward** [comment]
Unrelated test failure.

**Comment by jonatack** [comment]
For deprecation warnings, `AS::Deprecation::Reporting:68` adds missing periods to ends of sentences, I believe.

**Comment by vipulnsward** [comment]
Gah. Dumb of me. Anyway, will get others feedback if we want to just keep this now, else will remove those from this patch.

**Comment by jeremy** [comment]
I'd keep the periods you added. It feels strange to auto-add missing punctuation rather than properly punctuating the warnings in the first place. Turns out I [added this](https://github.com/rails/rails/commit/8fc97d198ef31c1d7a4b9b849b96fc08a667fb02), maybe to avoid updating the many warning messages. I'd prefer to see it go 😁

File: activesupport/lib/active_support/deprecation/reporting.rb [hunk 1]
```diff
@@ -65,7 +65,6 @@ def deprecated_method_warning(method_name, message = nil)
 
         def deprecation_message(callstack, message = nil)
           message ||= "You are using deprecated behavior which will be removed from the next major or minor release."
-          message += '.' unless message =~ /\.$/
```
Comment (vipulnsward) [inline activesupport/lib/active_support/deprecation/reporting.rb]: @jeremy done. Now making sure the sentence ends with '.' lies at where ever its defined.
Before https://github.com/rails/rails/commit/8fc97d198ef31c1d7a4b9b849b96fc08a667fb02, we relied on '.' being present in sentences.

**Comment by jeremy** [comment]
Would you squash the revert onto the other commit? Then we have a single commit that drops the period-addition behavior and adds missing periods.

**Comment by vipulnsward** [comment]
Squashed.

**Comment by jonatack** [comment]
Yes. Perhaps update the PR title to something like "Clean up warning messages and stop auto-adding periods."

**Pull request merged**